src/main/java/vazkii/botania/common/block/subtile/functional/SubTileRannuncarpus.java

```diff
@@ -78,7 +78,6 @@ public void onUpdate() {
 			int z = supertile.zCoord;
 
 			List<EntityItem> items = supertile.getWorldObj().getEntitiesWithinAABB(EntityItem.class, AxisAlignedBB.getBoundingBox(x - RANGE, y - RANGE_Y, z - RANGE, x + RANGE + 1, y + RANGE_Y, z + RANGE + 1));
-			List<ItemStack> itemFilter = SubTileHopperhock.getFilterForInventory(supertile.getWorldObj(), null,x,y-(supertile instanceof IFloatingFlower?1:2),z, false);
```

Comment:
Hmm, this was recently added in #60 but I guess this was not implemented properly?

Reply:
I would assume so. In the upstream 1.20.x implementation this List is used: https://github.com/VazkiiMods/Botania/blob/1.20.x/Xplat/src/main/java/vazkii/botania/common/block/flower/functional/RannuncarpusBlockEntity.java